pkg/apis/ceph.rook.io/v1/labels_test.go

```diff
@@ -58,8 +58,9 @@ func TestCephLabelsMerge(t *testing.T) {
 
 	// Merge with "all"
 	testLabels = LabelsSpec{
-		"all": {"allkey1": "allval1", "allkey2": "allval2"},
-		"mgr": {"mgrkey": "mgrval"},
+		"all":           {"allkey1": "allval1", "allkey2": "allval2"},
+		"mgr":           {"mgrkey": "mgrval"},
+		"detectversion": {"detect": "myversion"},
```

Comment:
In a greater sense, the issue is that the command reporter "cmdreporter" doesn't have labels/placement. The cmd reporter is used mostly for version detection, but it is also used for network inspection, and I think one more thing. Could we expand this to be for all uses of the command reporter?

Reply:
Yeah, I meant to look at that. The one other instance I see is for the networkreporter. I see it has a reference to the cephcluster, so it would be a simple change. Then what should we name it instead of `detectversion`? `cmdreporter` doesn't mean much to the user. What about `jobrunner`? Or maybe just go with `cmdreporter` to match the code where it's used.

Reply:
`cmdreporter` is set now